interfaces/builtin/x11.go

```diff
@@ -60,14 +137,52 @@ socket AF_NETLINK - NETLINK_KOBJECT_UEVENT
 bind
 `
 
+type x11Interface struct {
+	commonInterface
+}
+
+func (iface *x11Interface) AppArmorConnectedSlot(spec *apparmor.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
+	if !release.OnClassic {
+		old := "###PLUG_SECURITY_TAGS###"
+		new := plugAppLabelExpr(plug)
```

Comment:
`new` is a keyword in Go, although using it for variable names works, it does look a bit awkward. OTOH, we can do cosmetic cleanups after landing the code.

Reply:
I was just sticking to the convention that I saw in other files